algorithms/aqc/solving_qlsp/solving_qlsp_with_aqc.ipynb

```diff
@@ -0,0 +1,867 @@
+{
```

Comment:
<p>Maybe comment that this is a purely mathematical pre-process. That you do it here exact, however, in real life probably the condition number will be approximated somehow, or known in advance from some external considerations. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/688/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>